cg/constants/demultiplexing.py

```diff
@@ -29,6 +29,7 @@ class DemultiplexingDirsAndFiles(StrEnum):
     RUN_PARAMETERS_PASCAL_CASE: str = "RunParameters.xml"
     RUN_PARAMETERS_CAMEL_CASE: str = "runParameters.xml"
     SAMPLE_SHEET_FILE_NAME: str = "SampleSheet.csv"
+    CORRECT_SAMPLE_SHEET: str = "CorrectSampleSheet.csv"
```

Comment:
There is never a file generated called `CorrectSampleSheet.csv`. It is only there for testing purposes. And production code should not be adapted to suit the tests

Reply:
Good catch, I will move this